pkg/smokescreen/smokescreen.go

```diff
@@ -577,20 +577,20 @@ func checkIfRequestShouldBeProxied(config *Config, req *http.Request, outboundHo
 
 	switch action {
 	case EgressAclDecisionDeny:
-		decision.reason = "Role is not allowed to access this host"
```

Comment:
Previously, this would in certain cases this decision reason would be incorrect, which it should be something like ""Deny: User Configured".